client/blocks/login/two-factor-authentication/waiting-notification-approval.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import { localize } from 'i18n-calypso';
+
+class WaitingTwoFactorNotificationApproval extends Component {
+	render() {
+		return (
+			<div>
+				<form onSubmit={ this.onSubmitForm }>
+					<Card className="is-compact">
+						<p>
+							{ this.props.translate( 'We just sent a push notification to your WordPress mobile app. ' +
+								'Swipe or tap to open and verify your login.' ) }
+						</p>
+						<div>
+							<img className="two-factor-authentication__auth-code-preview" src="/calypso/images/login/pushauth.svg" />
+						</div>
+						<p>
+							{ this.props.translate( 'Or, type in the verification code generated by your Authenticator app' +
+								' or sent via text.' ) }
+						</p>
+					</Card>
+					<Card className="two-factor-authentication__form-action is-compact">
+						<p>
+							{ this.props.translate( 'Or, continue to your account using:' ) }
+						</p>
+						<p>
+							<a href="#">{ this.props.translate( 'A recovery code via text' ) }</a>
```

Comment:
Hi! I've found a possible matching string that has already been translated **40** times:
`translate( 'Send recovery code via text' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).